frontend/app/views/info/elevatedSupporter.scala.html

```diff
@@ -0,0 +1,21 @@
+@import com.gu.i18n.CountryGroup
+@import com.gu.memsub.subsv2.{CatalogPlan, MonthYearPlans}
+@import views.support.PageInfo
+@(  heroImage: model.OrientatedImages,
+    supporterPlans: MonthYearPlans[CatalogPlan.Supporter],
+    pageInfo: PageInfo,
+    detailImage: model.OrientatedImages)(implicit token: play.filters.csrf.CSRF.Token, countryGroup: CountryGroup)
+
+@main("Supporters", pageInfo=pageInfo, countryGroup=Some(countryGroup)) {
+
+    <section class="elevated-supporter">
+        @* ===== Become a Guardian Supporter ===== *@
```

Comment:
Are these comments useful? I reckon the fragment names provide sufficient self-documentation